### ISSUE TITLE
Do we need the clean gt for training?

### ISSUE THREAD

**Issue by stevewongv** [michaelofsbu/SpatialCorrection#2]
When I look into the code, I find that code relies on clean ground truth.
https://github.com/michaelofsbu/SpatialCorrection/blob/e2ea65d504d08eef35abbdacf882c8aa6caa45f4/main.py#L264
https://github.com/michaelofsbu/SpatialCorrection/blob/e2ea65d504d08eef35abbdacf882c8aa6caa45f4/utils/LabelClean.py#L46

Am I right?

**Comment by michaelofsbu** [comment]
@stevewongv Thank you for raising the issue. The clean ground truth of training set is only for evaluating the label correction performance. You could set `show_clean_performance=False` if the clean training label is not available. However, the current code may not fully support that (errors will rise). I will update it ASAP and let you know once it's done.

**Comment by michaelofsbu** [comment]
@stevewongv The code has been updated. You can easily get rid of the requirement for clean training labels by setting `show_clean_performance=False`.
